storage/src/tree_state.rs

```diff
@@ -1,8 +1,8 @@
 use hash::H256;
-use crypto::sha256_compress;
+use crypto::{sha256_compress, pedersen_hash};
 
 lazy_static! {
-	static ref EMPTY_ROOTS: Vec<H256> = [
+	static ref SPROUT_EMPTY_ROOTS: Vec<H256> = [
```

Comment:
`EMPTY_ROOS_32` or similar probably would be more convenient name, since this hardly relates to the sprout, just the tree height

Reply:
Oh, nevermind, sapling has non-zero initial leaves